python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -40,6 +40,12 @@ class MatcherSourceSelectorConfig(BaseSourceSelectorConfig):
         "(in the flux specified by sourceFluxType); <= 0 for no limit",
         default=40,
     )
+    excludeSaturated = pexConfig.Field(
+        dtype=bool,
+        doc="Exclude saturated objects using PixelFlags. For matchOptimisticB "
+            "set this to False to recover previous matcher selector behavior.",
+        default=True,
+    )
```

Comment:
The config parameter is called `excludeSaturated`, but it also rejects things that are on the edge and have an interpolated center value.  I think this config name should be changed.

Reply:
does `excludePixelFlags` or `excludeSatInterpEdge` work? Any better ideas?

Reply:
Of those `excludePixelFlags` is better and allows us to change the exact flags being excluded later.